src/Proto.Mailbox/Mailbox.cs

```diff
@@ -187,10 +192,20 @@ public void Start()
             }
             catch (Exception e)
             {
-                await _invoker.EscalateFailureAsync(e, msg);
+                HandleProcessMessageException(e, msg);
             }
             return true;
         }
+        
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I tried throwing an exception from `EscalateFailureAsync` and got the following stack trace, so it seems that it didn't inline:
```
System.Exception: Exception of type 'System.Exception' was thrown.
   at Proto.LocalContext.EscalateFailureAsync(Exception reason, Object message) in C:\code\github\protoactor-dotnet\src\Proto.Actor\LocalContext.cs:line 248
   at Proto.Mailbox.DefaultMailbox.HandleProcessMessageException(Exception exception, Object message) in C:\code\github\protoactor-dotnet\src\Proto.Mailbox\Mailbox.cs:line 205
   at Proto.Mailbox.DefaultMailbox.ProcessMessages() in C:\code\github\protoactor-dotnet\src\Proto.Mailbox\Mailbox.cs:line 150
```